library/compiler-builtins/compiler-builtins/build.rs

```diff
@@ -548,20 +548,28 @@ mod c {
             sources.extend(&[("__emutls_get_address", "emutls.c")]);
         }
 
+        // Optionally, link against a prebuilt compiler-rt library to supply
+        // optimized intrinsics instead of compiling a subset of compiler-rt
+        // from source.
+        let link_against_prebuilt_rt = env::var_os("LLVM_BUILTIN_RT_LIB").is_some();
```

Comment:
Maybe `LLVM_COMPILER_RT_LIB` to match `RUST_COMPILER_RT_ROOT`, and since this is probably linking the entire compiler-rt rather than just builtins. Also I believe the LLVM library is called "builtins" rather than "builtin"

Reply:
In the case of fedora, compiler-rt builtins is packaged as a distinct library. I plan on linking against it. Looking at `build.rs`, on some targets (e.x openbsd), is only a broader compiler-rt library available to link against?